spec/system/casework/viewing_an_application/application_details/client_other_charge_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+RSpec.describe 'Viewing the other charges for client in an application' do
+  include_context 'with stubbed application'
+
+  let(:application_data) do
+    JSON.parse(LaaCrimeSchemas.fixture(1.0).read).deep_merge(
+      'case_details' => {
+        'client_other_charge_in_progress' => client_other_charge_in_progress,
+        'partner_other_charge_in_progress' => partner_other_charge_in_progress,
+        'client_other_charge' => client_other_charge
+      }
+    )
+  end
+
+  let(:client_other_charge_in_progress) { nil }
+  let(:partner_other_charge_in_progress) { nil }
+  let(:client_other_charge) { nil }
+
+  before do
+    visit crime_application_path(application_id)
+  end
+
+  describe 'when no client other charge in progress' do
+    let(:client_other_charge_in_progress) { nil }
+
+    it 'does not show the card' do
+      expect(page).not_to have_css('h2.govuk-summary-card__title', text: 'Other charges')
+    end
+  end
+
+  describe 'when both client and partner other charge in progress' do
```

Comment:
This describe context suggests both will be 'yes', maybe best to reword to say if both questions have been answered